src/pages/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,10 @@
+import Header from '../../Components/Header/Header';
+export default function Dashboard() {
+  return (
+    <div>
```

Comment:
If you don't need this div (you are actually not passing any styles to it) replace it with React fragments.
Because you are rendering an unneeded DOM element.